models/market.js

```diff
@@ -0,0 +1,35 @@
+module.exports = (sequelize, type) => {
+    return sequelize.define('market', {
+      id: {
+        type: type.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      initialPurchaseValue: {
+        type: type.INTEGER,
+        defaultValue: 0,
+      },
+      immediatePurchaseValue: {
+        type: type.INTEGER,
+        defaultValue: 0,
+      },
+      finishDate: {
+        type: type.DATE,
+        allowNull: false,
+        validate: {
+            isDate: {
+                args: true,
+                msg: "Error: finishDate must be a date"
+            },
+            notEmpty: {
+                args: true,
+                msg: "Error: finishDate must not be empty"
```

Comment:
Si son mensajes que se envian al frontend, ps en español xd
```suggestion
                msg: "finishDate debe ser una fecha valida"
            },
            notEmpty: {
                args: true,
                msg: "finishDate debe ser una fecha valida"
```